mailer.http.class.php

```diff
@@ -218,14 +217,32 @@ protected function handle_response($response)
     protected function get_recipients()
     {
         $recipients = array();
+        $recipients_header_to = array();
 
         foreach ($this->to as $to) {
-            $recipients[] = array(
-                'address' => array(
-                    'email' => $to[0],
-                    'name' => $to[1]
-                ));
+            $recipients[] = $this->build_recipient($to[0], $to[1]);
+
+            // prepare for header_to
+            if(!empty($to[1])) {
+              $recipients_header_to[] = sprintf('%s <%s>', $to[1], $to[0]);
+            } else {
+              $recipients_header_to[] = $to[0];
+            }
+        }
+
+        // include bcc to recipients
+        // sparkposts recipients list acts as bcc by default
+        $bcc = $this->get_bcc(implode(',', $recipients_header_to));
+        if(!empty($bcc)) {
+          $recipients = array_merge($recipients, $bcc);
+        }
+
+        // include cc to recipients
+        $cc = $this->get_cc();
```

Comment:
this format works better
```
{
      address: {
        email: 'abc@xyz.com',
        header_to: 'Name 1<dddd@gmail.com>, Name 2 <eee@gmail.com>'
      }
    }

```

Specifying `name` attribute and also having full form of recipient (`name <email>`) in `header_to` creates weird structure.